src/components/UserPosts/UserPosts.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState, useEffect} from 'react'
+import { useMutation } from "@apollo/client";
+import * as gql from '../../queries/queries';
+
+const UserPosts = ({data, loading, error, userID}) => {
+  const [query, setQuery] = useState(1)
+  const [clicked, setClicked] = useState(false)
+  const [deletePost] = useMutation(gql.DELETE_POST, {
+    refetchQueries: [{ query: gql.GET_USER_POSTS(userID) }],
```

Comment:
Again, this is a real slick way to get the queries to refetch and display! Nice work.